src/doc/rustdoc/src/advanced-features.md

```diff
@@ -81,3 +81,10 @@ Then, when looking for it through the `rustdoc` search, if you enter "x" or
 "big", search will show the `BigX` struct first.
 
 There are some limitations on the doc alias names though: you can't use `"` or whitespace.
+
+You can add multiple aliases at the same time by using a list:
```

Comment:
"by using a list" seems potentially confusing because people will probably assume you mean `= ["one", "two"]`.

Reply:
I checked and there is no attribute using the `[]` syntax in the std. In addition, we give an example literally the line after, so I think there is very little possibility for a misunderstanding here.

Reply:
Good point :)